src/Game.js

```diff
@@ -0,0 +1,41 @@
+export default class Game {
+    constructor() {
+        // this.alive = true;
+        this.player1;
+        this.player2;
+        this.wordToGuess = '';
+        this.attemptedGuesses = []; // All of these will be displayed as guesses
+        this.maxWrongAttempts = 6;
+        this.wrongGuesses = 0;
+        this.correctGuesses = []; // These will be displayed as correct guesses only
+    }
+    reviewAttempt(guess) {
+        this.attemptedGuesses.push(guess);
+        if (!this.wordToGuess == guess || !this.wordToGuess.includes(guess)) {
+            this.wrongGuesses += 1;
+            this.wrongAttempt();
+        } else {
+            this.correctGuesses.push(guess);
+            this.correctAttempt();
+        }
+    }
+    wrongAttempt() {
+        if (this.wrongGuesses >= this.maxWrongAttempts) {
+            alert('The man is dead');
+            location.reload();
+        }
+    }
+    correctAttempt() {
+        let theWordSplit = this.wordToGuess.split('');
+        if (this.correctGuesses(-1) == this.wordToGuess) {
```

Comment:
`this.correctGuesses(-1)` should throw an error I think. `this.correctGuesses` is an Array, not a function, so that syntax isn't recognized.

try `this.correctGuesses[this.correctGuesses.length - 1]` or `this.correctGuesses.slice(-1)` if you want the last element. (this last way will return an array, however)

Reply:
the `.slice(-1)` thing is what I was going for, good call